Please edit/add entries, ideally in alphabetical order. For citations, I used google scholar cite, copied and pasted MLA, then deleted the title. 

| Name/link   | One-sentence description | Lead developer/s | Citation (MLA) | 
| :---    | :---------               | :----            | :----    |

| [coiaf](https://github.com/bailey-lab/coiaf) | Estimate sample-level MOIs using bi-allelic SNP data | Aris Paschalidis and Oliver Watson | Paschalidis, Aris, et al. bioRxiv (2022): 2022-05. |

| [COIL](https://github.com/kgalinsky/COIL) | Estimate sample-level MOIs using bi-allelic SNP data | Kevin Galinsky | Galinsky, Kevin, et al. Malaria journal 14.1 (2015): 1-9.|

| [dcifer](https://github.com/innager/dcifer) | Estimate relatedness between multiclonal samples using data on multi-allelic markers | Inna Gerlovina | Gerlovina, Inna, et al. Genetics 222.2 (2022): iyac126.|

| [DEploid](https://github.com/DEploid-dev/DEploid) || Joe Zhu | Zhu, Sha Joe, Jacob Almagro-Garcia, and Gil McVean. Bioinformatics 34.1 (2018): 9-15.|

| [estMOI](https://github.com/sammy-assefa/estMOI) | For a given sample, generate genome-wide MOI estimates from counts of micro-haplotypes spanned by paired-end reads generated by sequencing | Samuel Assefa | Assefa, Samuel A., et al. Bioinformatics 30.9 (2014): 1292-1294. |  

| [FreqEstimationModel](https://github.com/aimeertaylor/FreqEstimationModel) | Estimate population-level frequencies and prevalences of haplotype/genotypes spanning bi-allelic SNPs, and, based on the same SNPs, estimate the population-average MOI | Aimee Taylor | Taylor, Aimee. Diss. University of Oxford, 2016; Taylor, Aimee R., et al. Malaria journal 13.1 (2014): 1-11.| 

| [fwd-dream](https://github.com/JasonAHendry/fwd-dream) ||Jason Hendry | Hendry, Jason A., Dominic Kwiatkowski, and Gil McVean. PLoS computational biology 17.8 (2021): e1009287.|

| Hapster ||Curt Hansen| Hansen, Curt. Diss. UC Berkeley, 2015.|

| [HMMERTIME](https://github.com/nickbrazeau/HMMERTIME) || Nicholas Brazaeu ||

| [hmmIBD](https://github.com/glipsnort/hmmIBD) | Infer segments of identity-by-descent and relatedness between intra- and interpopulation monoclonal sample pairs | Stephen Schaffner | Schaffner, Stephen F., et al. Malaria journal 17.1 (2018): 1-4.| 

| [isoRelate](https://github.com/bahlolab/isoRelate)| Infer segments of identity-by-descent and relatedness between intrapopulation pairs of mono- and biclonal samples | Lyndal Henden | Henden, Lyndal, et al. PLoS genetics 14.5 (2018): e1007279. |

| [MALECOT](https://github.com/bobverity/MALECOT) | Estimate sample-level MOIs and population-level allele frequencies using bi-allelic SNP data (as in TheReaLMcCOIL), while also doing population assignment (as in STRUCTURE) | Robert Verity ||

| [McCOILR](https://github.com/OJWatson/McCOILR) | Rcpp wrapper for THE REAL McCOIL software| Oliver Watson ||

| [moimix](https://github.com/bahlolab/moimix) | An R package for sample-level MOI estimation using read count data, prinically written as an interface to flexmix::initFlexmix, which can be used to estimate an MOI as the number of components of a binomial mixture model, but can also be used to compute FWS, and prepare data for MOI estimation using COIL | Stuart Lee ||

| [moire](https://github.com/m-murphy/moire) | Estimate sample-level MOIs and population-level allele frequencies using data on multi-allelic markers| Maxwell Murphy | |

| [MultiLociBiallelicModel](https://github.com/Maths-against-Malaria/MultiLociBiallelicModel) | Estimate population-level frequencies and prevalences of haplotype/genotypes spanning bi-allelic SNPs, and, based on the same SNPs, estimate the population-average MOI | | Tsoungui Obama, Henri Christian Junior, and Kristan Alexander Schneider. "A maximum-likelihood method to estimate haplotype frequencies and prevalence alongside multiplicity of infection from SNP data." Frontiers in Epidemiology (2022): 34.|

| [paneljudge](https://github.com/aimeertaylor/paneljudge) | Estimate and simulate relatedness with uncertainty between intrapopulation monoclonal sample pairs   | Aimee Taylor | LaVerriere, Emily, et al. medRxiv (2021).| 

| [pfmix](https://github.com/cascobayesian/pfmix) ||Jack O'Brien||

| [THEREALMcCOIL](https://github.com/Greenhouse-Lab/THEREALMcCOIL) | Estimate sample-level MOIs and population-level allele frequencies using bi-allelic SNP data | Hsiao-Han Chang | Chang, Hsiao-Han, et al. PLoS computational biology 13.1 (2017): e1005348.|

| [SIMPLEGEN](https://github.com/mrc-ide/SIMPLEGEN) || Robert Verity ||
