Please edit/add entries, ideally in alphabetical order. For citations, I used google scholar cite, copied and pasted MLA, then deleted the title. 

| Name/link   | One-sentence description | Lead developer/s | Citation (MLA) | 
| :---    | :---------               | :----            | :----    |
| [dcifer](https://github.com/innager/dcifer) || Inna Gerlovina ||
| [DEploid](https://github.com/DEploid-dev/DEploid) || Joe Zhu | Zhu, Sha Joe, Jacob Almagro-Garcia, and Gil McVean. Bioinformatics 34.1 (2018): 9-15.|
| [FreqEstimationModel](https://github.com/aimeertaylor/FreqEstimationModel) | Infer MOIs and multi-SNP genotype frequencies from SNP-wise prevalence data on multiclonal samples | Aimee Taylor | Taylor, Aimee. Diss. University of Oxford, 2016; Taylor, Aimee R., et al. Malaria journal 13.1 (2014): 1-11.| 
| [fwd-dream](https://github.com/JasonAHendry/fwd-dream) ||Jason Hendry | Hendry, Jason A., Dominic Kwiatkowski, and Gil McVean. PLoS computational biology 17.8 (2021): e1009287.|
| Hapster ||Curt Hansen| Hansen, Curt. Diss. UC Berkeley, 2015.|
| [HMMERTIME](https://github.com/nickbrazeau/HMMERTIME) || Nicholas Brazaeu ||
| [hmmIBD](https://github.com/glipsnort/hmmIBD) | Infer segments of identity-by-descent and relatedness between intra- and interpopulation monoclonal sample pairs | Stephen Schaffner | Schaffner, Stephen F., et al. Malaria journal 17.1 (2018): 1-4.| 
| [isoRelate](https://github.com/bahlolab/isoRelate)| Infer segments of identity-by-descent and relatedness between intrapopulation pairs of mono- and biclonal samples | Lyndal Henden | Henden, Lyndal, et al. PLoS genetics 14.5 (2018): e1007279. |
| [MALECOT](https://github.com/bobverity/MALECOT) | Estimate sample-level MOIs and population-level allele frequencies using bi-allelic SNP data (as in The ReaLMcCOIL), while also doing population assignment (as in STRUCTURE) | Robert Verity ||
| [McCOILR](https://github.com/OJWatson/McCOILR) || Oliver Watson ||
| [moimix](https://github.com/bahlolab/moimix) | An R package for sample-level MOI estimation using read count data, prinically written as an interface to flexmix::initFlexmix, which can be used to estimate an MOI as the number of components of a binomial mixture model, but can also be used to compute FWS, and prepare data for MOI estimation using COIL | Stuart Lee ||
| [moire](https://github.com/m-murphy/moire) || Maxwell Murphy | |
| [paneljudge](https://github.com/aimeertaylor/paneljudge) | Infer and simulate relatedness with uncertainty between intrapopulation monoclonal sample pairs   | Aimee Taylor | LaVerriere, Emily, et al. medRxiv (2021).| 
| [pfmix](https://github.com/cascobayesian/pfmix) ||Jack O'Brien||
| [THEREALMcCOIL](https://github.com/Greenhouse-Lab/THEREALMcCOIL) | Estimate sample-level MOIs and population-level allele frequencies using bi-allelic SNP data | Hsiao-Han Chang | Chang, Hsiao-Han, et al. PLoS computational biology 13.1 (2017): e1005348.|
| [SIMPLEGEN](https://github.com/mrc-ide/SIMPLEGEN) || Robert Verity ||
